tests/test_auto_detection.py

```diff
@@ -37,6 +37,8 @@ def test_loop_auto():
     assert isinstance(policy, asyncio.events.BaseDefaultEventLoopPolicy)
     expected_loop = "asyncio" if uvloop is None else "uvloop"
     assert type(policy).__module__.startswith(expected_loop)
+    loop = asyncio.get_event_loop()
```

Comment:
Actually, is this alright?

Reply:
for future reference :
the change in auto_detection is to fix an unclosed socket ResourceWarning that appeared when I added the parameters
and rightly so since we dont close the loop in that test